src/components/Parent.vue

```diff
@@ -74,8 +83,17 @@ export default {
 
   // step 1 - mount component
   async mounted() {
-    // step 2 - get data from backend
-    await this.fetchData();
+    // step 2 - get data
+    if(isSecondaryWindow()){
+      // fetch data from indexedDB
+    const indexedDBData = await getData()
+    this.myJson = indexedDBData.selectionList
```

Comment:
if you make the changes in indextedDB-helper file then this will be `this.myJson = indexedDBData.dataList`